src/main/java/controller/UserAPI.java

```diff
@@ -100,7 +100,8 @@
         requestValidator.loginRequestValidator(data.keySet(), path);
         String username = data.get("Username").toString();
         String password = data.get("Password").toString();
-        long id = userAccessor.login(username, password);
+        String passwordHash = User.hashPassword(password);
```

Comment:
"passwordHash" is not serving any purpose.
Lets write: .login(username, User.hashPassword(password))

Reply:
Done.